CONTRIBUTING.md

```diff
@@ -99,3 +99,16 @@ In the Firestore emulator, create a collection called "nationalparks" and add a
 You can check the logs to see if it ran in the "Logs" tab of the emulator.
 
 Also, try querying App Search via the search endpoint: http://localhost:5001/nationalparks/us-central1/search?query=rocky
+
+## Publishing
+
+The npm script and extension should maintain the same version number and should always be published together to maintain compatibility.
+
+To publish the npm script:
+
+```
+# Update the version number in functions/package.json
+cd functions
+npm run build
```

Comment:
We could add a [prepublishOnly script to our package.json](https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts) to run `npm run build` automatically (this will also disallow publishing if that script exits with an error)

Reply:
That's a great idea.